installer/pyinstaller/samcli.spec

```diff
@@ -2,29 +2,38 @@
 import sys ; sys.setrecursionlimit(sys.getrecursionlimit() * 5)
 block_cipher = None
 exe_name = 'sam'
-analysis = Analysis(['../../samcli/__main__.py'],
-             binaries=[('/usr/local/lib/libcrypt.so.2', '.')],
-             datas=[],
-             hiddenimports=[],
-             hookspath=['./installer/pyinstaller'],
-             runtime_hooks=[],
-             excludes=[],
-             cipher=block_cipher)
+analysis = Analysis(
+    ['../../samcli/__main__.py'],
+    binaries=[
+        ('/usr/local/lib/libbz2.so.1', '.'),
+        ('/usr/local/lib/libcrypt.so.2', '.')
```

Comment:
There was an earlier discussion with @hnnasit about removing this all together. Do we know if it is possible to remove these custom configs just like the MacOS ones?

Reply:
Good point, we can try removing both of these to test. We can add them back easily anyways